assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -121,7 +121,7 @@ storage:
       mode: 0644
       contents:
         inline: |
-          KUBELET_IMAGE_URL=docker://k8s.gcr.io/hyperkube
```

Comment:
Why is this needed? The hyperkube images works just fine for now. I think we need a bit more thought in how to handle their deprecation.

Reply:
I have asked question about it in the description.